src/client/Microsoft.Identity.Client/TokenCache.ITokenCacheInternal.cs

```diff
@@ -19,6 +19,9 @@
 
 namespace Microsoft.Identity.Client
 {
+    /// <summary>
+    /// IMPORTANT: this class is perf critical; any changes must be benchmarked using Microsoft.Identity.Test.Performace
+    /// </summary>
     public sealed partial class TokenCache : ITokenCacheInternal
```

Comment:
This is what JM was referring to, can we have an aka.ms link or something which directs us to how to run the tests, which information is needed before/after, and where to put it...etc... thank you.

Reply:
We don't have a place to record historical info, @pmaytak recorded some data (see PR description). For a historical view, we need the same machine. We don't have a stable "golden config" machine. 

So the proposal is to just run the benchmark test (it's just a console app) manually - before any changes and after any changes.

Reply:
right, i understand that. but what JM is saying, is, for future reference, how do we run the performance runner? like someone new on the team, how do they run it? what are they looking for? We need to have this info somewhere, shouldn't be too hard to do. it's just a short doc on how to run it and what to look for, either in the wiki or in the onenote. we can add to it later, but this should be part of the PR, i think.

Reply:
later, he can figure out how to handle the historical data, but that's separate.

Reply:
Done.

Reply:
Done.